lib/bamboo/adapters/local_adapter.ex

```diff
@@ -30,13 +30,12 @@ defmodule Bamboo.LocalAdapter do
 
   @behaviour Bamboo.Adapter
 
-  @doc "Adds email to `Bamboo.SentEmail` and opens it in new browser tab"
+  @doc "Adds email to `Bamboo.SentEmail`, can automatically open it in new browser tab"
   def deliver(email, %{open_email_in_browser_url: open_email_in_browser_url}) do
     %{private: %{local_adapter_id: local_adapter_id}} = SentEmail.push(email)
     open_url_in_browser("#{open_email_in_browser_url}/#{local_adapter_id}")
   end
 
-  @doc "Adds email to `Bamboo.SentEmail`"
```

Comment:
Add of curiosity, why remove the doc? Could we revert changes to this?

Reply:
Elixir 1.11 throws a warning for this behavior. You can't add two different calls to `@doc` for the same function, you have to condense it into a single place.